p5js/06_FastStyleTransfer_Mirror/sketch.js

```diff
@@ -0,0 +1,57 @@
+/* ===
+ML5 Example
+06_Fast_Style_Transfer_Mirror
+Fast Style Transfer Mirror Example with p5.js
+This uses a pre-trained model of The Great Wave off Kanagawa and Udnie (Young American Girl, The Dance)
+=== */
+
+let fastStyle;
+let video;
+let modelReady = false;
+let cameraReady = false;
+let startPredict = false;
+let resultImg;
+
+function setup() {
+  createCanvas(300, 300).parent('canvasContainer');
+  pixelDensity(1);
+  background(0);
+  video = createCapture(VIDEO, cameraLoaded);
+  resultImg = createImg('');
+  resultImg.hide();
+  video.size(200, 200);
+  video.hide();
+  fastStyle = new ml5.FastStyle('models/udnie', modelLoaded);
+}
+
+function draw(){
+  image(resultImg, 0, 0, 300, 300);
+}
+
+function cameraLoaded() {
+  cameraReady = true;
+}
+
+function modelLoaded() {
+  modelReady = true;
+}
+
+function togglePredicting() {
+  startPredict = !startPredict;
+  if(startPredict){
+    select('#controlBtn').html('Stop');
+    predict();
+  } else {
+    select('#controlBtn').html('Start');
+  }
+
+}
+
+function predict() {
+  if(cameraReady && modelReady && startPredict) {
+    const result = fastStyle.transfer(video.elt);
```

Comment:
Just a reminder for us to maybe sit down and talk about https://github.com/ml5js/ml5-library/issues/27 sometime. These examples might (emphasis on the *might*) be cleaner with `loadImage()` and returning back to the user a `p5.Image`.